scripts/ci-frontend-metrics.sh

```diff
@@ -2,6 +2,7 @@
 set -e
 
 ERROR_COUNT="$(./node_modules/.bin/tsc --project tsconfig.json --noEmit --strict true | grep -oP 'Found \K(\d+)')"
+ACCESSIBILITY_ERROR_COUNT="$(yarn -s test:accessibilty --json | jq '.errors')"
```

Comment:
Be careful wrapping node binaries in yarn. It can cause the commands to exit with code 2. See this https://github.com/grafana/grafana/pull/35722#issuecomment-864108017.

Reply:
Ah, interesting. Thanks for this